test/Microsoft.ML.Tests/TextLoaderTests.cs

```diff
@@ -795,5 +796,77 @@ public void TestTextLoaderKeyTypeBackCompat()
                 Assert.True(result.Schema[featureIdx].Type is KeyDataViewType keyType && keyType.Count == typeof(uint).ToMaxInt());
             }
         }
+
+        public class IrisNoFields
+        {
+        }
+
+        public class IrisPrivateFields
+        {
+            [LoadColumn(0)]
+            private float SepalLength;
+
+            [LoadColumn(1)]
+            private float SepalWidth { get; }
+
+            public float GetSepalLenght()
+                => SepalLength;
+
+            public void SetSepalLength(float sepalLength)
+            {
+                SepalLength = sepalLength;
+            }
+        }
+        public class IrisPublicGetProperties
+        {
+            [LoadColumn(0)]
+            public float SepalLength { get; }
+
+            [LoadColumn(1)]
+            public float SepalWidth { get; }
+        }
+
+        public class IrisPublicProperties
+        {
+            [LoadColumn(0)]
+            public float SepalLength { get; set; }
+
+            [LoadColumn(1)]
+            public float SepalWidth { get; set; }
+        }
+
+        [Fact]
+        public void TestTextLoaderNoFields()
+        {
+            var dataPath = GetDataPath(TestDatasets.irisData.trainFilename);
+            var mlContext = new MLContext();
+
+            // Class with get property only.
+            var dataIris = mlContext.Data.CreateTextLoader<IrisPublicGetProperties>(separatorChar: ',').Load(dataPath);
```

Comment:
Could you update the doc of `CreateTextLoader<T>` to explain what is allowed?

Reply:
Will do!